spk/postgresql/Makefile

```diff
@@ -0,0 +1,24 @@
+SPK_NAME = postgresql
+SPK_VERS = 11.6
+SPK_REV = 1
+SPK_ICON = src/postgresql.png
+
+DEPENDS = cross/$(SPK_NAME)
+
+MAINTAINER = DigitalBox
+DESCRIPTION = PostgreSQL is a powerful, open source object-relational database system with over 30 years of active development that has earned it a strong reputation for reliability, feature robustness, and performance
+
+RELOAD_UI = yes
+DISPLAY_NAME = PostgreSQL
+STARTABLE = yes
+
+HOMEPAGE   = http://www.postgresql.com
+LICENSE    = PostgreSQL Licence
+
+WIZARDS_DIR = WIZARD_UIFILES
+
+INSTALLER_SCRIPT = src/installer.sh
+SSS_SCRIPT = src/start-stop-status.sh
```

Comment:
I propose to take benefits from generic installer support and service control methods described at: https://github.com/SynoCommunity/spksrc/wiki/Service-Support#installer

If you want to look at code, you can refer to:
- https://github.com/SynoCommunity/spksrc/blob/master/mk/spksrc.service.installer
- https://github.com/SynoCommunity/spksrc/blob/master/mk/spksrc.service.start-stop-status

I have no idea how long we may go on supporting DSM 5 but using these support allow package to run as-is there too.

Reply:
My bad, it was OK for my previous Icecast submission, so I will mimick that one to be OK :)